05-14 20:13:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ошибка ввода/вывода: "IOException reading /data/my_planets.csv"
IO Exception: "IOException reading /data/my_planets.csv"; SQL statement:
CREATE TABLE PLANETS (ID INT AUTO_INCREMENT, NAME VARCHAR(35), TEXTURE VARCHAR(45), RADIUS FLOAT, DISTANCE FLOAT, ORB_SPEED FLOAT, ROT_SPEED FLOAT, TILT FLOAT, PRIMARY KEY(ID)) AS SELECT * FROM CSVREAD('/data/my_planets.csv') [90028-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.tools.Csv.convertException(Csv.java:603)
	at org.h2.tools.Csv.read(Csv.java:199)
	at org.h2.expression.Function.getValueForColumnList(Function.java:2551)
	at org.h2.table.FunctionTable.<init>(FunctionTable.java:67)
	at org.h2.command.Parser.readTableFilter(Parser.java:1212)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1859)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:1968)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1853)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1674)
	at org.h2.command.Parser.parseSelect(Parser.java:1662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5866)
	at org.h2.command.Parser.parseCreate(Parser.java:4122)
	at org.h2.command.Parser.parsePrepared(Parser.java:351)
	at org.h2.command.Parser.parse(Parser.java:306)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.prepareCommand(Parser.java:243)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:172)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at updateDB.main(updateDB.java:21)
Caused by: java.io.FileNotFoundException: \data\my_planets.csv (Системе не удается найти указанный путь)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at org.h2.store.fs.FilePathDisk.newInputStream(FilePathDisk.java:311)
	at org.h2.store.fs.FileUtils.newInputStream(FileUtils.java:207)
	at org.h2.tools.Csv.initRead(Csv.java:321)
	at org.h2.tools.Csv.readResultSet(Csv.java:221)
	at org.h2.tools.Csv.read(Csv.java:197)
	... 20 more
05-14 20:18:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "PLANETS" уже существует
Table "PLANETS" already exists; SQL statement:
CREATE TABLE PLANETS (ID INT AUTO_INCREMENT, NAME VARCHAR(35), TEXTURE VARCHAR(45), RADIUS FLOAT, DISTANCE FLOAT, ORB_SPEED FLOAT, ROT_SPEED FLOAT, TILT FLOAT, PRIMARY KEY(ID)) AS SELECT * FROM CSVREAD('d:/my_planets.csv') [42101-178]
05-14 20:32:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "UPDATE PLANETS AS SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; ожидалось "identifier"
Syntax error in SQL statement "UPDATE PLANETS AS SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; expected "identifier"; SQL statement:
update planets as SELECT * FROM CSVREAD('d:/my_planets.csv') [42001-178]
05-14 20:37:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "UPDATE PLANETS AS SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; ожидалось "identifier"
Syntax error in SQL statement "UPDATE PLANETS AS SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; expected "identifier"; SQL statement:
update planets as SELECT * FROM CSVREAD('d:/my_planets.csv') [42001-178]
05-14 20:38:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "UPDATE PLANETS AS SET SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; ожидалось "SET"
Syntax error in SQL statement "UPDATE PLANETS AS SET SELECT[*] * FROM CSVREAD('d:/my_planets.csv') "; expected "SET"; SQL statement:
update planets as set SELECT * FROM CSVREAD('d:/my_planets.csv') [42001-178]
05-14 20:41:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Синтаксическая ошибка в выражении SQL "UPDATE PLANETS AS SET NAME[*] = 'SELECT name FROM CSVREAD('D:/MY_PLANETS.CSV')' "; ожидалось "SET"
Syntax error in SQL statement "UPDATE PLANETS AS SET NAME[*] = 'SELECT name FROM CSVREAD('D:/MY_PLANETS.CSV')' "; expected "SET"; SQL statement:
update planets as set name = 'SELECT name FROM CSVREAD('d:/my_planets.csv')' [42001-178]
05-14 20:59:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "NAME" не найден
Column "NAME" not found; SQL statement:
SELECT name FROM CSVREAD('d:/my_planets.csv') [42122-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:148)
	at org.h2.command.dml.Select.prepare(Select.java:831)
	at org.h2.command.Parser.prepareCommand(Parser.java:248)
	at org.h2.engine.Session.prepareLocal(Session.java:442)
	at org.h2.engine.Session.prepareCommand(Session.java:384)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:75)
	at updateDB.main(updateDB.java:20)
05-14 21:00:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "ITEMS" не найдена
Table "ITEMS" not found; SQL statement:
UPDATE items SET name = ?, texture = ?, radius = ?, distance = ?, orb_speed = ?, rot_speed = ?, tilt = ? WHERE id = ? [42102-178]
05-14 21:00:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-178]
05-14 21:00:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-178]
05-14 21:00:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Объект уже закрыт
The object is already closed [90007-178]
05-15 21:08:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "PLANETS" не найдена
Table "PLANETS" not found; SQL statement:
SELECT * FROM PLANETS [42102-178]
05-15 21:09:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "PLANETS" не найдена
Table "PLANETS" not found; SQL statement:
DROP TABLE PLANETS [42102-178]
05-15 21:46:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
CREATE TABLE IF NOT EXISTS PLANETS (ID INT AUTO_INCREMENT, NAME VARCHAR(35), TEXTURE VARCHAR(45), RADIUS FLOAT, DISTANCE FLOAT, ORB_SPEED FLOAT, ROT_SPEED FLOAT, TILT FLOAT, PRIMARY KEY(ID)) AS SELECT * FROM CSVREAD('d:/Projects/Eclipse/SOL/data/planets_table_DBUpdate.csv') [21002-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.message.DbException.get(DbException.java:143)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at updateDB.main(updateDB.java:74)
05-15 21:47:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
CREATE TABLE IF NOT EXISTS PLANETS (ID INT AUTO_INCREMENT, NAME VARCHAR(35), TEXTURE VARCHAR(45), RADIUS FLOAT, DISTANCE FLOAT, ORB_SPEED FLOAT, ROT_SPEED FLOAT, TILT FLOAT, PRIMARY KEY(ID)) AS SELECT * FROM CSVREAD('d:/Projects/Eclipse/SOL/data/planets_table_DBUpdate.csv') [21002-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.message.DbException.get(DbException.java:143)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at updateDB.main(updateDB.java:74)
05-15 21:48:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Неверное количество столбцов
Column count does not match; SQL statement:
CREATE TABLE IF NOT EXISTS PLANETS (ID INT AUTO_INCREMENT, NAME VARCHAR(35), TEXTURE VARCHAR(45), RADIUS FLOAT, DISTANCE FLOAT, ORB_SPEED FLOAT, ROT_SPEED FLOAT, TILT FLOAT, PRIMARY KEY(ID)) AS SELECT * FROM CSVREAD('D:\Projects\Eclipse\SOL\data\planets_table_DBUpdate.csv') [21002-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.message.DbException.get(DbException.java:143)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:117)
	at org.h2.command.CommandContainer.update(CommandContainer.java:79)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:160)
	at updateDB.main(updateDB.java:74)
05-16 00:15:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Недопустимое значение "0" для параметра "columnIndex"
Invalid value "0" for parameter "columnIndex" [90008-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:227)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3165)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3193)
	at org.h2.jdbc.JdbcResultSet.getString(JdbcResultSet.java:286)
	at project.test.updateDB.main(updateDB.java:27)
05-16 00:16:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Нет данных
No data is available [2000-178]
05-16 00:17:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Нет данных
No data is available [2000-178]
05-16 13:57:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Столбец "orb-speed" не найден
Column "orb-speed" not found [42122-178]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:344)
	at org.h2.message.DbException.get(DbException.java:178)
	at org.h2.message.DbException.get(DbException.java:154)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3122)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3210)
	at org.h2.jdbc.JdbcResultSet.getFloat(JdbcResultSet.java:706)
	at project.test.updateDB.<init>(updateDB.java:47)
	at project.test.updateDB.main(updateDB.java:20)
09-07 13:22:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "INFO" не найдена
Table "INFO" not found; SQL statement:
SELECT * FROM info [42102-181]
09-07 13:23:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "INFO" не найдена
Table "INFO" not found; SQL statement:
SELECT * FROM info [42102-181]
09-07 13:23:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "MOONS" не найдена
Table "MOONS" not found; SQL statement:
SELECT * FROM MOONS [42102-181]
09-08 12:19:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "INFO" не найдена
Table "INFO" not found; SQL statement:
SELECT * FROM info [42102-181]
09-08 12:19:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Таблица "MOONS" не найдена
Table "MOONS" not found; SQL statement:
SELECT * FROM MOONS [42102-181]
